pages/doc/wavefront_spring_boot3.md

```diff
@@ -0,0 +1,450 @@
+  ```
+  wavefront.freemium-account=true
+  ```
+    {% include tip.html content="If you don’t want Wavefront to auto-negotiation a freemium account for you, set the value to `false`."%}
+1. Restart your application.
+
+<a name="proxy"></a>
+<p><span style="font-size: large; font-weight: 500">Use the Wavefront Proxy</span></p>
+
+The [Wavefront proxy](proxies.html) ingests data and forwards them to the Wavefront service in a secure, fast, and reliable manner. It prevents data loss, simplifies firewall configuration, and allows you to filter or enrich data before they arrive at the Wavefront service.
+{% include note.html content="Supported with Wavefront Proxy version 7.0 and later. Before sending data via the proxy, you need to [Install and Manage Wavefront Proxies](proxies_installing.html)."%}
+
+Copy and paste the following property to the `application.properties` file.
+```
+management.wavefront.uri=proxy://<Proxy_Host>:2878
+```
+
+<p><span style="font-size: large; font-weight: 500">Specify Application and Service Names</span></p>
+
+If you have more than one Spring Boot application, you can specify the names of the application and the service in the `application.properties` file.
+
+{{site.data.alerts.important}}
+<p>The valid characters in an application and service name are: a-z, A-Z, 0-9, hyphen ("-"), underscore ("_"), dot ("."), forward slash ("/") and comma (","). </p>
+<p>If your application or service names have any other characters other than the valid characters, the Wavefront service replaces each of those characters with a hyphen ("-").</p>
+{{site.data.alerts.end}}
+
+<br/>Example:
+```
+management.wavefront.application.name=my-application
+management.wavefront.application.service-name=my-service
+```
+
+Example: If you are using a YAML file.
+```
+management:
+  wavefront:
+    application:
+      name: my-application
+      service-name: my-service
+```
+
+Optionally:
+* If you configured `spring.application.name` in your application, it is automatically used as the service name.
+
+You can configure the cluster and shard names under the `management.wavefront.application` namespace as well. This information is used to tag metrics and traces.
+* If you want to take full control over [`ApplicationTags`](trace_data_details.html#application-tags), you can create a `@Bean`.
+* If you want to customize the instance that is auto-configured, add an `ApplicationTagsBuilderCustomizer` bean.
+
+
+## Wavefront Spring Boot Integration
+
+If you already have a Wavefront account, you can start the setup and examine the dashboards from the Wavefront Spring Boot integration. This is supported for [Wavefront trial accounts](https://tanzu.vmware.com/observability).
+
+1. Log in to your Wavefront instance.
+2. Select **Integrations**, search for Spring Boot, and select the Spring Boot integration.
```

Comment:
2. Click **Integrations** on the toolbar, search for Spring Boot, and click the Spring Boot integration tile.